### PR TITLE
1.8

### PR DESCRIPTION
**Pull request by DeltaDizzy**

### CODE REVIEW

File: main/HazardousBody.md [hunk 1]
```diff
@@ -54,9 +55,10 @@ Body
 
 |Property|Format|Description|
 |--------|------|-----------|
-|heat|Double|The average heat on the body.|
-|interval|Single|How much time passes between applying the heat to a vessel.|
-|AltitudeCurve|FloatCurve|A multiplier of the average heat that gets applied at a certain altitude.|
-|LatitudeCurve|FloatCurve|A multiplier of the average heat that gets applied at a certain latitude.|
-|LongitudeCurve|FloatCurve|A multiplier of the average heat that gets applied at a certain longitude.|
-|HeatMap|File Path|A greyscale map for fine control of the heat on the body. Black = 0, White = 1.|
+|sumTemp|Boolean|LOOK AT CODE/ASK SIGMA|
```
Comment (democat3457) [inline main/HazardousBody.md]: What? Will this be filled out or just as a placeholder?

File: index.md [hunk 2]
```diff
@@ -36,12 +36,13 @@ subtitle: A mod to modify the planetary system used by KSP
   + Ocean subnode
     - Fog subnode
   + [HazardousBody subnode]({{ site.baseurl }}{% link main/HazardousBody.md %})
-  + [Particles subnode]({{ site.baseurl }}{% link main/Particles.md %})
   + [Debug subnode]({{ site.baseurl }}{% link main/Debug.md %})
+  #### [WARNING | The Particles subnode has been removed in Kopernicus 1.8.0-1, with no replacement. Only use this node if you pack is restricted to KSP 1.7.3 or earlier.]
+  + [Particles subnode]({{ site.baseurl }}{% link main/Particles.md %})
```
Comment (democat3457) [inline index.md]: Can you ~~strikethrough~~ Particles so further clarification?

**Review by democat3457 — CHANGES_REQUESTED** [review]

File: main/HazardousBody.md (hunk 1 above)
Comment (DeltaDizzy) [reply main/HazardousBody.md]: Yes that will be filled out, It is `Whether or not to add the ambientTemp to the calculated temperature.`

Comment (democat3457) [reply main/HazardousBody.md]: Are you changing it or should I?

**Pull request merged**